.github/README_es.md

```diff
@@ -208,7 +208,7 @@ func main() {
 
 ```
 
-#### 📖 [**Middleware & Next**](https://docs.gofiber.io/api/ctx#next)
+#### 📖 [**Middleware y Más**](https://docs.gofiber.io/api/ctx#next)
```

Comment:
The previous translation was correct, considering that it talks about the `middleware` and the `Next` method. (Besides, "Next" in spanish is not "Más")

Reply:
Reverted to Next, just translated "&" to "y".